public/stylesheets/ers_styles.css

```diff
@@ -39,12 +37,12 @@ div.grid-layout {
 }
 
 
-.validation-message {font-weight:bold; border-left: 4px solid #df3034; padding-left:14px; font-size:19px; margin-top:1em; margin-left: -1em;}
+/*.validation-message {font-weight:bold; border-left: 4px solid #df3034; padding-left:14px; font-size:19px; margin-top:1em; margin-left: -1em;}
```

Comment:
Should we be removing unused css instead of commenting? A couple of others in this file too

Reply:
yup, missed removing it when i was going back over it

Reply:
removed commented out css